File: Algorithm/Sorting/Insertion sort.cpp
#include<iostream>
using namespace std;
int main(){
    
    int size; cin>>size;
    int arr[size];
    
    for(int& x: arr) cin>>x;
    
    for(int i=1;i<size;i++){
        int temp=arr[i],index=i;
        for(int j=i-1;j>=0;j--){
            if(arr[j]>temp){
                swap(arr[j],arr[index]);
                index--;
            }
        }
    }
    
    for(int x: arr)
    cout<<x<<" ";
    
}

File: Algorithm/Sort/Insertion Sort.py
size=int(input(""))
arr=list(map(int,input("").split()))
for i in range(1,size):
    key=arr[i]
    index=i
    for j in range(i-1,-1,-1):
        if key<=arr[j]:
            arr[index]=arr[j]
            index-=1
    arr[index]=key
# print(arr)

for i in arr:
    print(i,end=" ")